tokens/space/size.yaml

```diff
@@ -0,0 +1,31 @@
+size:
```

Comment:
can we call this `length` instead?

`size` could also be `font-size`

Reply:
Could length confusing if this value is used for say, padding top? I'm not sure. It seems like it would be okay. Agreed that "size" is confusing. I'm all for aligning with the CSS values.

Reply:
`padding-top` (and, it's preferred logical counterpart `padding-block-start`, takes a `<length>` value:
https://developer.mozilla.org/en-US/docs/Web/CSS/padding-top#syntax

Reply:
AFAIK, `size` was intended for sizing steps like in `font-size`. For `padding` and `margin` we'd have `spacer` (rather than `length`) which is set to a slightly different scale.